core/src/com/unciv/models/translations/Translations.kt

```diff
@@ -196,6 +219,10 @@ val eitherSquareBraceRegex = Regex("""\[|\]""")
 // Analogous as above: Expect a {} pair with any chars but } in between and capture that
 val curlyBraceRegex = Regex("""\{([^}]*)\}""")
 
+// Analogous as above: Expect a <> pair with any chars but > in between and capture that
+val pointyBraceRegex = Regex("""\<([^>]*)\>""")
```

Comment:
The inspections should complain that <> do not need escaping??

Reply:
Tbh, regex is complete magic to me and I just copy+pasted things until it worked. At my end Android Studio is complaining about every single of these regexes, so I just ignored it and hoped that either nobody would mind or someone else would later fix it :)

Reply:
> hoped that nobody would mind

What an invitation for satire!

Let's just be glad that it doesn't refuse to _compile_ a perfectly valid Regex (again).